### DISCUSSION TITLE
GitHub integration: Ability to hide deployments

### DISCUSSION THREAD

**Discussion by digitalclubb** [vercel/community#4143] [Help] [answered]
We're in the process of moving our microfrontend monorepo to Vercel and manage deploys with the GitHub integration.

There is a lot of noise generated for each project that is deployed when we only care about one (our entry point to our application).

See the following as an example:
![Screenshot 2023-09-19 at 09 08 19](https://github.com/vercel/community/assets/2639929/07c066ee-f411-4a7d-91aa-f39c2cac087c)

17 projects are deployed and we only care about seeing 'main' as a preview URL. main is our application shell which imports our other applications from their respective URLs. We still need the other projects to be deployed as they are imported into main.

All of these deploys also create a lot of noise in the GitHub timeline which means comments are often lost or end up become buried:
![Screenshot 2023-09-19 at 09 08 29](https://github.com/vercel/community/assets/2639929/e6c9206f-1a89-429f-bc68-2389c0748892)

I'd love the ability, via the Vercel.json configuration, to have an option to silence deploys.

```
{
	"silentDeploys": true
}
```

Perhaps you have a table of the deploys you want to show, in our case main, and then have a collapsed section with the other deploys (similar to ignored) so you can still check your other projects have built ok.

Possibly related: https://github.com/orgs/vercel/discussions/601

**Comment by cjayyy** [comment, marked as answer]
I did get following answer from Vercel support when asking about disabling these comments;

> You aren't able to disable those notifications mentioned from within Vercel as they aren't comments by the vercel-bot but in fact notifications added by GitHub regarding any related deployments activity. If this is something you would like to remove I would recommend reaching out to GitHub directly as it is part of their Deployment Activity: https://docs.github.com/en/repositories/viewing-activity-and-data-for-your-repository/viewing-deployment-activity-for-your-repository
